### PR TITLE
feat: add metadata for pypi

### PR DESCRIPTION
**Pull request by Sanix-Darker**
## WHAT

Just added metadatas for pypi

### CODE REVIEW

File: server/setup.cfg [hunk 1]
```diff
@@ -1,6 +1,11 @@
 [metadata]
 name = weaverbird
 version = 0.12.3
+description = "A visual data pipeline builder with various backends"
+authors = ["Toucantoco <tokar@gmail.com>"]
```
Comment (PrettyWood) [inline server/setup.cfg:5]:
```suggestion
authors = ["Toucan Toco <dev@toucantoco.com>"]
```

File: server/setup.cfg [hunk 2]
```diff
@@ -1,6 +1,11 @@
...
+keywords = ["mongodb", "pandas", "sql_translator", "data", "dataviz", "pipeline"]
+readme = "README.md"
+license = "BSD 3-Clause"
```
Comment (PrettyWood) [inline server/setup.cfg:8]:
```suggestion
license = "BSD-3-Clause"
```

**Review by PrettyWood — CHANGES_REQUESTED** [review]
thanks

**Comment by fspot** [comment]
Changes included in https://github.com/ToucanToco/weaverbird/pull/1348 :ok_hand:

**Pull request closed without merging**